uniconnect-epfl/uniconnect
# fix(graph): fixed issue on android and non-deterministic tests

# What I did
I fixes the nodes' display problem on android and fixed non-deterministic tests.

# How I did it
- I encapsulated the ClipPath in a Group component from React-Native-SVG so that it updates the states on Android.
- I removed hard coded timeouts in timers and just used wait for.

# How to verify it
- Go to Android Emulator and try dragging the nodes / the graph, the profile pictures should follow along.
- Run the tests

# Pre-merge checklist
The changes I have introduced:
